rskj-core/src/main/java/co/rsk/validators/BlockTimeStampValidationRule.java

```diff
@@ -46,8 +46,22 @@ public boolean isValid(Block block) {
         boolean result = blockTime - currentTime <= this.validPeriodLength;
         if(!result) {
             logger.warn("Error validating block. Invalid timestamp {}.", blockTime);
-            panicProcessor.panic("invalidtimestamp", String.format("Error validating block. Invalid timestamp %d.", blockTime));
         }
         return result;
     }
+
+    @Override
+    public boolean isValid(Block block, Block parent) {
+        boolean result = this.isValid(block);
```

Comment:
should we validate the parent too?

Reply:
just realized that we're validating that the parent comes before the child, so it's implicitly validated.